.github/workflows/build.yml

```
@@ -0,0 +1,162 @@
+        if: github.event_name == 'pull_request'
+        with:
+          name: pr
+          path: pr/
+
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 10
+
+      - name: Get commit branch and commit message from push
+        if: github.event_name == 'push'
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)